public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -251,6 +251,7 @@ export const DashNav = React.memo<Props>((props) => {
         onChangeTimeZone={updateTimeZoneForSession}
         onRefreshClick={() => trackDashboardsToolbarActionsClicked('refresh')}
         onZoomClick={() => trackDashboardsToolbarActionsClicked('zoom_out_time_range')}
+        onTimePickerClick={() => trackDashboardsToolbarActionsClicked('time_picker')}
```

Comment:
this is what I was talking about. I think DashNav shouldn't know what to pass to that track function, in order to not be coupled. 

`onTimePickerClick={trackToolbarTimePickerClicked}

export function trackToolbarTimePickerClicked() {
  reportInteraction(shareAnalyticsEventNames.dashboardsToolbarActionsClicked, { item: 'time_picker' });
}
`

Reply:
Make sense. I created one function for each of the options

Reply:
and just a nit.shouldn't it be `trackToolbarTimePickerClick`instead of `trackToolbarTimePickerClicked` ?